pkg/cmd/protoc-gen-gogoroach/main.go

```diff
@@ -14,8 +14,35 @@ import (
 	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 	"github.com/gogo/protobuf/vanity"
 	"github.com/gogo/protobuf/vanity/command"
+	"regexp"
+	"strings"
 )
 
+var builtinRegex *regexp.Regexp = regexp.MustCompile("github.com/cockroachdb/cockroach/pkg/(?P<capture>(bytes|context|encoding/binary|errors|fmt|io|math|github\\.com|(google\\.)?golang\\.org)([^a-z]|$$))")
```

Comment:
I'm not really sure with what was happening originally, but assuming you've cracked it, a comment would be helpful.

Reply:
I don't fully understand exactly how this works, but I added a comment explaining what I DO know. :)